art_utils/src/rosparam_services.py

```diff
@@ -1,24 +1,26 @@
 #!/usr/bin/env python
 import rospy
-from art_msgs.srv import getARTableConfig, getARTableConfigResponse
-from art_msgs.msg import ARTableConfig
+from art_msgs.srv import GetRosparam, GetRosparamResponse
 
 
 class RosparamServices:
     def __init__(self):
-        self.get_art_conf_srv = rospy.Service('/art/conf/get', getARTableConfig, self.art_rosparam_conf_get_cb)
-        rospy.spin()
+        self.get_rosparam_service = rospy.Service('/rosparam_get', GetRosparam, self.get_rosparam_cb)
+
+    def get_rosparam_cb(self, req):
+        try:
+            param = rospy.get_param(req.param_name)
+        except KeyError:
+            param = "ERROR: Parameter [" + req.param_name + "] is not set"
```

Comment:
Pro účel vrácení chyby by asi bylo lepší mít v tom servisu nějaký flag jestli se parametr povedlo získat, nebo ne. Vracet chybu v poli pro data se mi nezdá úplně šťastné.

Reply:
Ale to je asi jen detail...

Reply:
předělám

Reply:
A ještě mě napadá - místo převodu na string by tam spíš měl být převod na json, ne? Tj. ```json.dumps(param)```. Není to to samé, viz https://stackoverflow.com/questions/47648886/difference-between-strdict-and-json-dumpsdict.

Reply:
Dík, tohle mě nenapadlo.